bug_report.php

```diff
@@ -74,12 +75,29 @@
 require_api( 'string_api.php' );
 require_api( 'user_api.php' );
 require_api( 'utility_api.php' );
+require_api( 'api_token_api.php' );
 
-form_security_validate( 'bug_report' );
+# Check if this is a usual submission via UI or scripted autosubmission
```

Comment:
I'm totally fine to replace option of reporting via bug_report.php with option of reporting via REST. It is even better.

However, I would like to have crash API present in both SOAP and REST.